src/page/login/login.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {Props} from 'react';
+import {View, Text, TextInput, TouchableHighlight} from 'react-native';
+import {styles} from './loginCss';
+import Reactotron from 'reactotron-react-native';
+// import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+// import {faCoffee} from '@fortawesome/free-solid-svg-icons';
+// import Logo from '../../resource/images/logo.svg';
+
+const Login: React.FC = (Props: Props) => {
```

Comment:
`props: Props`
and you should type Props by your self

type Props = {
  foo: () => void
  bar: string
}